scripts/pi-hole/js/network.js

```diff
@@ -66,17 +60,17 @@ function mixColors(ratio, rgb1, rgb2)
 
 $(document).ready(function() {
     tableApi = $("#network-entries").DataTable( {
-        "rowCallback": function( row, data, index )
+        "rowCallback": function( row, data )
         {
-             var color, mark, lastQuery = parseInt(data["lastQuery"]);
+             var color, mark, lastQuery = parseInt(data.lastQuery);
```

Comment:
I see that a lot (maybe even the majority) of changes in this PR is
```diff
- some["a"]
+ some.a
```

Is there any advantage in this change else than using two characters less?

Reply:
https://eslint.org/docs/rules/dot-notation

Also, in the past it was more perfromant in some cases.